src/shell/neoshell.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="CommandDotNet" Version="7.0.2" />
```

Comment:
are we using this?

Reply:
Thank you for catching this! This was part of the initial experiment. It's no long used.